db/schema.rb

```diff
@@ -133,6 +134,15 @@
     t.index ["nomis_staff_id"], name: "index_pom_details_on_nomis_staff_id"
   end
 
+  create_table "responsibilities", force: :cascade do |t|
```

Comment:
looks like this got added by accident - no harm done in this case though.

Reply:
Good catch, will take it out pending POM290 :)